lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -387,15 +387,15 @@ computeTxSize proxy action cs =
 
     dummyKeyHashRaw = BS.pack (replicate 28 0)
 
-    withdrawals = mkWithdrawals
+    wdrls = mkWithdrawals
```

Comment:
frankly speaking like more previous naming, but understand the probable rationale (80-line styling)

Reply:
Yeah, I usually try to keep naming consistent with the surrounding context.